package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "ya-done-appium",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "description": "Ready to use yadda - appium - chai framework",
   "repository": "https://github.com/britishgas-engineering/ya-done-appium",
-  "author": "Darren Corbett",
+  "author": "Biplav",
```

Comment:
author: "Centrica" or ignore the field

Reply:
I was thinking about what will be better as people will moving here and there.